wiki/BBCode/en.md

```diff
@@ -1,581 +1,417 @@
+```
+
+### List
+
+This will format the text intro a list.
+
+In BBCode, there are two parts for making the list: the wrapper and the bullets.
+Where the bullets are placed inside the wrapper.
+
+- Dedicated buttons:
+  - List: ![List button](img/list.jpg "List button")
+  - List type: ![List Type button](img/list-type.jpg "List Type button")
+  - List bullet: ![List Bullet button](img/list-bullet.jpg "List Bullet button")
+- Notes:
+  - All bullets must be surrounded by the list/list type tags.
+  - These are the vaild arguments for `TYPE`:
+    - _(empty)_ - bulleted
+    - `1` - numbered
+    - `a` - lettered (lowercased)
+    - `A` - lettered (upper-cased)
+    - `i` - roman numeral (lowercased)
+    - `I` - roman numeral (upper-cased)
+  - By default, bulleted lists uses sqaure bullets.
+  - Lists inside lists are known to be buggy.
+- Syntax:
+  - Where `TYPE` is one of the listed types above.
+  - If `TYPE` is not defined, bullets will be used instead.
+```
+[list=TYPE]
+[*]...
+[/list]
+```
+
+### Image
+
+This will add images to your post.
+The images can be sourced from anywhere, just as long as the image exists from the given url.
+
+It is a nooby mistake to try to link your images directly from your PC.
+Please understand, but using `C:\Users\Alice\Pictures\got-to-see-this.jpg` will **not** work!
+Instead, you must upload it to the internet using a third party application.
+
+Such applications could be: [imgur](https://imgur.com/) or [puush](https://puush.me/).
+
+- Dedicated button: ![Image button](img/image.jpg "Image button")
+- Notes:
+  - If you have many images or large images, it is recommended to put them inside a [Box](#box).
+- Syntax:
+  - Where `LINK` is a direct link to an image.
+```
+[img]LINK[/img]
+```
+
+### Youtube
+
+This embeds a YouTube video in your post.
+
+- Dedicated button: ![Image button](img/youtube.jpg "Image button")
+- Notes:
+  - If you have many images or large images, it is recommended to put them inside a [Box](#box).
+- Syntax:
+  - Where `YT-ID` is the YouTube video ID (11 characters long), **not** the whole url!
+    - The YouTube video ID is located after the `?v=` url parameter.
+```
+[youtube]YT-ID[/youtube]
+```
+
+### Heading (v1)
+
+This adds a big fancy pink heading.
+This is usually used to denote a new section in yor post.
+
+- Dedicated button: _none_
+- Notes:
+  - You must type the syntax yourself.
+- Syntax:
+```
+[heading]...[/heading]
+```
+
+### Heading (v2)
+
+This adds a big fancy purple heading with a horizontal line.
+This is usually used to denote a new section in yor post.
+
+- Dedicated button: _none_
+- Notes:
+  - This only works in the beatmaps fourm!
+  - This only appears after postingm not in preview.
+  - You must type the syntax yourself.
+- Syntax:
+```
+[...]
+```
+
+### Notice
+
+This places the paragraph inside a white box.
+
+- Dedicated button: _none_
+- Notes:
+  - You must type the syntax yourself.
+- Syntax:
+```
+[notice]
+...
+[/notice]
+```
+
+## Trivia
```

Comment:
hmm... this is more of a "here is some useless, but interesting things" -- there is a "Notes" section at the very top that explains certain quirks/behavior about BBCode in the forums.